gallery/src/components/NxBinaryDonutChart/NxBinaryDonutChartCustomExample.scss

```diff
@@ -9,6 +9,6 @@
     display: block;
     height: 45px;
     .nx-binary-donut-chart__arc {
-      stroke: var(--nx-swatch-purple-70);
+      stroke: var(--nx-swatch-purple-70) !important;
```

Comment:
```suggestion
      stroke: var(--nx-swatch-purple-70);

      @include nx-dark-mode-helpers.dark-mode {
        stroke: var(--nx-swatch-purple-70);
      }
```

Reply:
Oh right, forgot about this :sweat_smile: